andela/ah-orcas-frontend
# #162751889 redesign list articles ui

#### What does this PR do?
```redesign view articles page ```
#### Description of Task to be completed?
```redesign article listing to make it look more pleasing ```
#### How should this be manually tested?
- git clone this repo
```
$ git clone https://github.com/andela/ah-orcas-frontend.git
```
- Navigate to this branch
```
git checkout ft-redesign-list-articles-162751889
```
- Install the node modules
```
$ npm install
```
- Start the server
```
$ npm start
```
- It should open your browser and navigate to http://localhost:3000
- Marvel at the awesomeness

#### What are the relevant pivotal tracker stories?
[#162751889](https://www.pivotaltracker.com/story/show/162751889)

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-21 at 10 33 58" src="https://user-images.githubusercontent.com/8406752/50330365-fa157480-050b-11e9-9938-0fb2f730846a.png">

#### Checklist:
